CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Added
+- Start publishing a container image of kubectl-gs as giantswarm/kubectl-gs
```

Comment:
Should be `as quay.io/giantswarm/kubectl-gs?, right?

Reply:
Longterm we decided dockerhub to be the source of truth. `crsync` will move the image there.

@JosephSalisbury - correct me if I'm wrong.

Reply:
@kopiczko it's the other way around, right? we push everything to `quay` (still), and crysnc moves it to dockerhub and ACR?

Reply:
yeah, see the second diagram here: https://intranet.giantswarm.io/docs/architecture-specs-adrs/specs/registry-mirror/#how-are-images-synchronised-between-registries

Reply:
Yes but we want dockerhub to be the SoT eventually. So why would be advertise quay?

Reply:
ah, with you. yeah, `giantswarm/kubectl-gs` makes sense as our source of truth